src/Rector/PHPUnit/ShouldCallParentMethodsRector.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DrupalRector\Rector\PHPUnit;
+
+use PhpParser\Node;
+use PHPStan\Analyser\Scope;
+use Rector\Core\Rector\AbstractScopeAwareRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+class ShouldCallParentMethodsRector extends AbstractScopeAwareRector
+{
+    public function getNodeTypes(): array
+    {
+        return [
+            Node\Stmt\ClassMethod::class,
+        ];
+    }
+
+    /**
+     * @phpstan-param Node\Stmt\ClassMethod $node
+     *
+     * @param \PhpParser\Node         $node
+     * @param \PHPStan\Analyser\Scope $scope
+     *
+     * @return \PhpParser\Node|null
+     */
+    public function refactorWithScope(Node $node, Scope $scope)
+    {
+        if ($scope->getClassReflection() === null) {
+            return null;
+        }
+
+        if (!$scope->getClassReflection()->isSubclassOf(\PHPUnit\Framework\TestCase::class)) {
+            return null;
+        }
+
+        $parentClass = $scope->getClassReflection()->getParentClass();
+
+        if ($parentClass === null) {
+            return null;
+        }
+
+        $parentClassesNames = $scope->getClassReflection()
+            ->getParentClassesNames();
+        if (!in_array(\PHPUnit\Framework\TestCase::class, $parentClassesNames)) {
+            return null;
+        }
```

Comment:
Doesn't the subclass check beforehand catch this? Or is it part of what was copy pasted

Reply:
I was scared for not finding deeper subclasses, but didn't test without this. Will run a quick test

Reply:
Seems you are actually right :x Also added a test that shows it doesn't remove it without a testcase as parent